docker-compose.prod.yml

```
@@ -0,0 +1,50 @@
+      - "8080:8080"
+    volumes: [] # 배포
+    stdin_open: true
+    tty: true
+    environment:
+      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
+      - JWT_SECRET_KEY=${JWT_SECRET_KEY}
+
+    depends_on:
+      - mysql
+    env_file:
+        - .env
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)